rancher/rancher
# [2.8] bump rancher-wins to `v0.4.17`

## Issue: https://github.com/rancher/rancher/issues/46220
 
## Problem

Maintenance bump for rancher-wins to resolve several CVEs

## Solution

 
## Testing
This version of wins has bumped the system-agent to `v0.3.7`, which includes changes for the `agentTLSMode` feature. This feature was not completely implemented for windows nodes. To ensure there are no regressions with that feature enabled, a full regression check should be done both with and without the `agentTLSMode` feature enabled. 

## Engineering Testing
### Manual Testing
I've built this version of wins and used it locally to deploy clusters, I did not see any issues provisioning and upgrading custom clusters. 

### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * Validation (Go Framework)
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

## QA Testing Considerations

 
### Regressions Considerations
Ensure the `agentTLSMode` feature functions as expected when using this version of wins. 

Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_
